### PR TITLE
[VT-18] - add distance to navigation data

### PR DESCRIPTION
**Pull request by mateuszRybczonek**
This PR add additional data to `Navigation Data` component about distance made by the vessel since last report.

screen shot 2018-06-10 at 16 35 46

### CODE REVIEW

File: src/components/__tests__/NavigationData.spec.js [hunk 1]
```diff
@@ -19,4 +20,20 @@ describe('NavigationData.vue', () => {
   it('renders BaseList component', () => {
     expect(wrapper.findAll(BaseList)).toHaveLength(1)
   })
+
+  it('renders proper number of items', () => {
+    expect(wrapper.findAll('[data-test-base-list-item]')).toHaveLength(3)
+  })
+
+  it('contains Course item', () => {
+    expect(wrapper.text()).toContain('90°')
```
Comment (vivensio) [inline src/components/__tests__/NavigationData.spec.js]: What do you think of asserting `${report.course}°` here? That way, the values are not bound to this line, but to the real thing we're testing.

File: src/components/__tests__/NavigationData.spec.js [hunk 2]
```diff
@@ -19,4 +20,20 @@ describe('NavigationData.vue', () => {
...
+  })
+
+  it('contains Speed item', () => {
+    expect(wrapper.text()).toContain('12 kn')
```
Comment (vivensio) [inline src/components/__tests__/NavigationData.spec.js]: same

File: src/utils/coordinates-utils.js [hunk 3]
```diff
@@ -31,18 +31,18 @@ function decimalToDMS (decimal, latitude = true, degrees = 0, minutes = 0, secon
...
+  const dlng = toRadians(arrLng - depLng)
 
   const a = Math.sin(dlat / 2) * Math.sin(dlat / 2) +
     Math.cos(lat1) * Math.cos(lat2) *
     Math.sin(dlng / 2) * Math.sin(dlng / 2)
 
   const c = 2 * Math.atan2(Math.sqrt(a), Math.sqrt(1 - a))
 
-  return R * c
+  return (R * c / 1852).toFixed(0)
```
Comment (vivensio) [inline src/utils/coordinates-utils.js]: I don't know this math here. Why are we doing `/ 1852`? Just curious. :)

Comment (mateuszRybczonek) [reply src/utils/coordinates-utils.js]: to get Nautical miles from meters :)

File: src/components/NavigationData.vue [hunk 4]
```diff
@@ -49,13 +49,18 @@
       },
       fetchingReports: {
         type: Boolean
+      },
+      distanceMadeSinceLastReport: {
```
[the remark this answers is not in the document]
Comment (vivensio) [reply src/components/NavigationData.vue]: whoa! Didn't know about that, Also, who am I telling?! 🙈Thanks for your reply. :)

File: src/utils/coordinates-utils.js (hunk 3 above)
Comment (vivensio) [reply src/utils/coordinates-utils.js]: 1 nm = 1852 m. Good to know. 😃

**Pull request merged**